src/editor/EditorStatusBar.js

```diff
@@ -237,6 +257,22 @@ define(function (require, exports, module) {
         _updateOverwriteLabel(event, editor, newstate, true);
         editor.toggleOverwrite(newstate);
     }
+
+    /**
+     * Update LineEndings indicator
+     * @param {Event} event (unused)
+     */
+    function _updateLineEndings(event) {
+        var editor = EditorManager.getActiveEditor();
+        var document = editor.document;
+
+        // update label with no transition
+        _updateLineEndingsLabel(event, editor, true);
+        // Update the line ending in the document if needed(label is already updated to new value).
+        if(document.getLineEndings() !== $statusLineEndings.text()) {
```

Comment:
Relying on the UI label may be unsafe, if it's translated to something else in any locale